cha-ching-app/src/ElementsProviderCheckout.js

```diff
@@ -4,22 +4,24 @@ import CheckoutForm from "./CheckoutForm";
 import Error from "./Error";
 import Loading from "./Loading";
 import { useLocation } from "react-router-dom";
+import { renderIf } from "./utils";
 
 const stripePromise = loadStripe(process.env.REACT_APP_PUBLIC_STRIPE_KEY);
 const ElementsProviderCheckout = () => {
   const isEmpty = (object) => Object.keys(object).length === 0;
-  const { state: options } = useLocation();
+  const { state: options = {} } = useLocation();
```

Comment:
this looks like a safety to prevent breakage, makes me feel we could also start implementing ramda functions for double safety!